packages/react-kit/src/components/productCard/ProductCard.tsx

```diff
@@ -62,31 +60,30 @@ const Wrapper = ({
   }
   return <>{children}</>;
 };
+
 export const PhygitalLabel = ({ ...rest }) => {
   return <TopLeftRibbon {...rest} data-text="Phygital" />;
 };
+
 export const ProductCard = (props: IProductCard) => {
   const {
-    asterisk = false,
-    avatar,
-    onAvatarError,
     avatarName,
-    bottomText,
     currency,
     dataCard = "product-card",
     dataTestId = "offer",
     imageProps,
     isHoverDisabled = false,
-    onAvatarNameClick,
     onCardClick,
     price,
     productId,
     title,
     tooltip = "",
     tooltipProps = {},
-    productType
+    CTAOnHover
```

Comment:
where is the prop to hide the animation on hover the image? on roblox only the CTA is animated, no the image/video

Reply:
It's old props: `isHoverDisabled`